tests/ui/create_dir.rs

```diff
@@ -0,0 +1,13 @@
+// run-rustfix
+#![allow(unused_must_use)]
+#![warn(clippy::create_dir)]
+
+fn not_create_dir() {}
```

Comment:
What does this test case do?

```suggestion
fn create_dir() {}
```

would test that non-std `create_dir` isn't linted.

Reply:
This reflects my intention nicer, thanks :)